llvm/lib/Target/AMDGPU/Disassembler/AMDGPUDisassembler.cpp

```diff
@@ -345,6 +345,25 @@ static DecodeStatus decodeOperand_VSrcT16_Lo128(MCInst &Inst, unsigned Imm,
                               (AMDGPU::OperandSemantics)OperandSemantics));
 }
 
+template <AMDGPUDisassembler::OpWidthTy OpWidth, unsigned ImmWidth,
+          unsigned OperandSemantics>
+static DecodeStatus
+decodeOperand_VSrcT16_Lo128_Deferred(MCInst &Inst, unsigned Imm,
```

Comment:
Looks this can be combined with decodeOperand_VSrcT16_Lo128(), e.g., by adding a `Deferred` template parameter?

Reply:
I think it can be combined but I think it's better to seperate them. It's a naming convension in this file that the deferred decoder has the defered name on the function